### PR TITLE
chore: Adds mentor infos for Inkdpixels

### PR DESCRIPTION
**Pull request by Inkdpixels**

### CODE REVIEW

File: mentors/javascript/i.json [hunk 1]
```diff
@@ -1 +1,8 @@
-[]
+[{
+  "github_username": "Inkdpixels",
+  "name": "Tyll Weiß",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I love JavaScript for for it's flexibility and fast-paced community. I've written and maintained quite a bit of TypeScript projects, both personally and professionally."
+}]
```
Comment (rpottsoh) [inline mentors/javascript/i.json]: Very simple formatting change please.  
```json
[
  {
    "github_username": "Inkdpixels",
    "name": "Tyll Weiß",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "I love JavaScript for for it's flexibility and fast-paced community. I've written and maintained quite a bit of TypeScript projects, both personally and professionally."
  }
]
```

File: mentors/typescript/i.json [hunk 2]
```diff
@@ -1 +1,8 @@
-[]
+[{
+  "github_username": "Inkdpixels",
+  "name": "Tyll Weiß",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I love TypeScript for for it's advance in features and type-safety, all while still being close to Javascript. I've written and maintained quite a bit of TypeScript projects, both personally and professionally."
+}]
```
Comment (rpottsoh) [inline mentors/typescript/i.json]: Like the other json file same formatting change.
```json
[
  {
    "github_username": "Inkdpixels",
    "name": "Tyll Weiß",
    "link_text": null,
    "link_url": null,
    "avatar_url": null,
    "bio": "I love TypeScript for for it's advance in features and type-safety, all while still being close to Javascript. I've written and maintained quite a bit of TypeScript projects, both personally and professionally."
  }
]
```

File: mentors/javascript/i.json (hunk 1 above)
Comment (Inkdpixels) [reply mentors/javascript/i.json]: Done :)

File: mentors/typescript/i.json (hunk 2 above)
Comment (Inkdpixels) [reply mentors/typescript/i.json]: Done :)

**Pull request merged**